pkg/timeutils/utils_test.go

```diff
@@ -36,9 +52,37 @@ func TestGetTimestamp(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		o := GetTimestamp(c.in, now)
+		o, err := GetTimestamp(c.in, now)
 		if o != c.expected {
-			t.Fatalf("wrong value for '%s'. expected:'%s' got:'%s'", c.in, c.expected, o)
+			if err != nil {
```

Comment:
Are there any error tests we want to include? perhaps the tests you removed - like "0" ?